src/hiveboard_bridge/test/SocketToMoveByIntegrationTest.cpp

```diff
@@ -69,10 +68,28 @@ int main(int argc, char** argv) {
     MessageDTO moveByMessageDTO(1, 2, moveByRequestDTO);
 
     // Send the message over TCP to the bridge
+#pragma clang diagnostic push
+#pragma ide diagnostic ignored "EndlessLoop"
```

Comment:
hmmmm whats that? you can disable clang with comments instead of pragma. You could probably just put a bool that's true instead and it wouldn't mind

Reply:
some automatic stuff that CLion put after I right-clicked on something to mute the warning about an endless loop. I'll lose that